<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddBookWindow</class>
 <widget class="QDialog" name="AddBookWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>697</width>
    <height>657</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Название:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="name"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Количество:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="count"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_9">
       <item>
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>Номер</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="idLE"/>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="1" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string>Раздел: </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="subsectionBook">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">Margin-raight: 3%;</string>
       </property>
       <item>
        <property name="text">
         <string>Художественная литература после 1917</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Художественная литература до 1917</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Зарубежная литература</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Детская литература</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Учебный фонд</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Ценный фонд</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="2" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Автор</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="authorLE"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QPushButton" name="searchAuthor">
         <property name="text">
          <string>Поиск</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="addAuthor">
         <property name="text">
          <string>Добавить</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="3" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Результат поиска:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QListWidget" name="resultSearchAuthors"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_6">
       <item>
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Авторы книги:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QListWidget" name="bookAuthors"/>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="4" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_7">
       <item>
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>Издатель:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="publisherLE"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_8">
       <item>
        <widget class="QPushButton" name="searchPublisher">
         <property name="text">
          <string>Поиск</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="addPublisher">
         <property name="text">
          <string>Добавить</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="5" column="0" rowspan="3" colspan="2">
    <layout class="QVBoxLayout" name="verticalLayout_11"/>
   </item>
   <item row="6" column="1">
    <widget class="QLabel" name="label_9">
     <property name="text">
      <string>Результат поиска:</string>
     </property>
    </widget>
   </item>
   <item row="7" column="1">
    <widget class="QListWidget" name="resultSearchPublisher"/>
   </item>
   <item row="8" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout_7">
     <item>
      <widget class="QLabel" name="label_10">
       <property name="text">
        <string>Издатель книги: </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="bookPublisher">
       <property name="text">
        <string>Не выбрано</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>37</width>
         <height>17</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="9" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="save">
       <property name="text">
        <string>Сохранить</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
